src/misc/cache/lv_cache.c

```diff
@@ -52,7 +52,11 @@ lv_cache_t * lv_cache_create(const lv_cache_class_t * cache_class,
     cache->size = 0;
     cache->ops = ops;
 
-    cache->clz->init_cb(cache);
+    if(cache->clz->init_cb(cache) == false) {
```

Comment:
if(!cache->clz->init_cb(cache))

Reply:
I think false will be more clear.
I trend to show the compare value clearly.